files/ini_file.py

```diff
@@ -186,10 +186,10 @@ def do_ini(module, filename, section=None, option=None, value=None, state='prese
                                 else:
                                     index = index + 1
                         break
-                else:
-                    # comment out the existing option line
+                elif state == 'absent':
```

Comment:
Looking through most of the module the state seems explicit everywhere else, thought I'd just continue on with that, I think it makes humanly reading through the code much simpler.